src/logger/loggerApi.c

```diff
@@ -1189,7 +1188,7 @@ static void sendTimerConfig(struct Serial *serial, size_t startIndex, size_t end
         json_uint(serial, "st", 0, 1);
         json_uint(serial, "mode", cfg->mode, 1);
         json_float(serial, "alpha", cfg->filterAlpha, FILTER_ALPHA_PRECISION, 1);
-        json_uint(serial, "ppr", cfg->pulsePerRevolution, 1);
+        json_float(serial, "ppr", cfg->pulsePerRevolution, 2, 1);
```

Comment:
is 2 decimal points adequate here? wondering if this should be 4 or 6 

1.333333? :)